test/markdownlint-cli2-config-option-with-extends/viewme.md

```diff
@@ -0,0 +1,3 @@
+# Title
```

Comment:
Please use exact copies of the same-named file in other tests.

Reply:
done